### PR TITLE
 [release/3.1] Switch to consolidated Windows build images and Svc pool

### PR DESCRIPTION
**Pull request by dougbu**
- includes a fake Arcade update matching recent release/3.x changes

### CODE REVIEW

**Review by MattGal — APPROVED** [review]
Given the challenges with Arcade 3.1, It's OK to update the eng/common folder too; the arcade update will just bring the same changes when we unblock it.

**Review by jonfortescue — APPROVED** [review]

**Review by wtgodbe — APPROVED** [review]

**Comment by dougbu** [comment]
Now includes a fake Arcade update that matches recent release/3.x changes. We don't need `windows.vs2019.amd64` in our parts of the build but Arcade apparently does.

File: src/Servers/Kestrel/shared/test/TestConstants.cs [hunk 1]
```diff
@@ -8,6 +8,6 @@ namespace Microsoft.AspNetCore.Testing
     public class TestConstants
     {
         public const int EOF = -4095;
-        public static TimeSpan DefaultTimeout = TimeSpan.FromSeconds(30);
+        public static TimeSpan DefaultTimeout = TimeSpan.FromMinutes(1);
```
Comment (dougbu) [inline src/Servers/Kestrel/shared/test/TestConstants.cs:11]: @JamesNK @Tratcher @halter73 @BrennanConroy this seemed painless to me and is definitely the simplest option. But please let me know if you object to such a general change for what's mostly a single test failing.

**Comment by dougbu** [comment]
Reset approvals to remind myself I want to hear from @JamesNK @Tratcher @halter73 @BrennanConroy

File: src/Servers/Kestrel/shared/test/TestConstants.cs (hunk 1 above)
Comment (JamesNK) [reply src/Servers/Kestrel/shared/test/TestConstants.cs:11]: My preference is to increase the timeout on the individual test. But if that isn't possible then I don't have a problem with this.

Perhaps add a comment to say what test the 1 minute timeout is for?

Comment (dougbu) [reply src/Servers/Kestrel/shared/test/TestConstants.cs:11]: I was just saving myself some copying and pasting from `TaskTimeoutExtensions` and `TestConstants`. Given your preference, I'll switch to the focused approach after dinner or tomorrow morning.

**Review by Tratcher — APPROVED** [review]

**Pull request merged**